src/ECER.Tests/Integration/Utilities/DataverseSdk/QueryTests.cs

```diff
@@ -24,6 +24,8 @@ public class QueryTests : IAsyncLifetime
     Guid.Parse("40188b8a-9fa7-4a2f-b065-c4f9dd3b0fdf")
   ];
 
+  private static string applicationId = "36ae9353-28b6-4c15-827f-2fe14a609e70";
```

Comment:
I'm hesitant with hard coding the applicationId, cause doesn't dynamics wipe out our data on the daily? So this test will eventually fail.